EIPS/eip-draft_AUTH_USURP.md

```diff
@@ -0,0 +1,81 @@
+
+We know that EOAs cannot provide ideal user experience or safety, and there is a desire in the community to change the norm to contract-based accounts, but if that transition is designed without regard for the vast majority of users today—for whom Ethereum has always meant EOAs—we will be continually struggling against the need to support both of these userbases. This EIP provides a path not to enshrine EOAs, but to provide a migration path off of them, once and for all.
+
+This proposal combines well with, but is distinct from, [EIP-3074](./eip-3074.md), which provides opcodes that could enable any externally owned account (EOA) to delegate its signing authority to an arbitrary smart contract. It allows an EOA to authorize a contract account to act on its behalf _without forgoing its own powers_, while this EIP provides a final migration path off the EOA's original signing key.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+### Conventions
+
+  - **`top - N`** - the `N`th most recently pushed value on the EVM stack, where `top - 0` is the most recent.
+  - **invalid execution** - execution that is invalid and must exit the current execution frame immediately, consuming all remaining gas (in the same way as a stack underflow or invalid jump).
+  - **empty account** - account where its balance is 0, its nonce is 0, and it has no code.
+
+### `AUTHUSURP` (`0xf8`)
+
+A new opcode `AUTHUSURP` shall be created at `0xf8`. It shall take two stack elements and return one stack element.
+
+#### Input
+
+| Stack     | Value        |
+| --------- | ------------ |
+| `top - 0` | `offset`     |
+| `top - 1` | `length`     |
+
+#### Output
+
+| Stack      | Value     |
+| ---------- | --------- |
+| `top - 0`  | `address` |
+
+#### Behavior
+
+`AUTHUSURP` behaves identically to `CREATE` (`0xf0`), except as described below:
+
+  - If `authorized` (as defined in EIP-3074) is unset, execution is invalid.
+  - If `authorized` points to an empty account, then `static_gas` remains 32,000. Otherwise, `static_gas` shall be 7,000.
+  - `AUTHUSURP` does not check the nonce of the `authorized` account.
+  - The initcode runs at the address `authorized`.
+  - If the initcode returns no bytes, its execution frame must be reverted, and `AUTHUSURP` returns zero.
+  - After executing the initcode, but before the returned code is deployed, if the account's code is non-empty, the initcode's execution frame must be reverted, and `AUTHUSURP` returns zero.
+  - The code is deployed into the account with the address `authorized`.
+
```

Comment:
Are transactions and signatures signed with the original private key still valid and able to be submitted to the blockchain, or should they revert?